src/map.cpp

```diff
@@ -781,7 +781,8 @@ vehicle *map::move_vehicle( vehicle &veh, const tripoint &dp, const tileray &fac
     // If not enough wheels, mess up the ground a bit.
     if( !vertical && !veh.valid_wheel_config() && !( veh.is_watercraft() && veh.can_float() ) &&
         !veh.is_flying_in_air() && dp.z == 0 ) {
-        veh.velocity += veh.velocity < 0 ? 2000 : -2000;
+        veh.velocity -= veh.velocity < 0 ?
+                        std::max( veh.velocity, -2000 ) : std::min( veh.velocity, 2000 );
```

Comment:
May be like so? Should do the same thing but less of a brain teaser
```suggestion
        veh.velocity -= std::clamp(veh.velocity, -2000, 2000); // extra drag
```